File: indigo-frontend/src/app/domain/author.ts
export class Author {

    constructor(
        public id:string,
        public name:string,
        public sort?: string,
        public description?: any,
        public provider?: string,
        public image?:string,
        public numBooks?: number,
        public coverUrl?: string,
        public originalImage?: string
        ) {
    }
}

File: indigo-backend/src/main/java/com/martinia/indigo/author/domain/model/Author.java
package com.martinia.indigo.author.domain.model;

import com.martinia.indigo.common.domain.model.NumBooks;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.io.Serializable;
import java.util.Date;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Author implements Serializable {

	private String id;
	private String name;
	private String sort;
	private String description;
	private String provider;
	private String image;
	private NumBooks numBooks;
	private Date lastMetadataSync;

}
